### PR TITLE
Sp3 frontend auth

### PR DESCRIPTION
**Pull request by Spellbreeze**
Collaborators: Xander, Parkaranjot

### CODE REVIEW

File: Civility Optics/Networking/NetworkingService.swift [hunk 1]
```diff
@@ -356,15 +362,15 @@ extension NetworkingService {
...
           }
           catch {
               print(error)
           }
           
-          completion(post)
+          completion(user)
           
 ////        let result = try? JSONDecoder().decode([UserResult].self, from: data)
 ////          completion(result?.first?.name)
```
Comment (madhura112) [inline Civility Optics/Networking/NetworkingService.swift:376]: don't need these

**Review by madhura112 — APPROVED** [review]
looks great

File: Civility Optics/Networking/NetworkingService.swift [hunk 2]
```diff
@@ -413,6 +422,129 @@ extension NetworkingService {
...
+        
+        guard checkStatus(res) else {
+          return
+        }
+        
+        printResponse(data)
+      }.resume()
+    }
+    
+
```
Comment (parkaransingh) [inline Civility Optics/Networking/NetworkingService.swift:510]: good job on the merge conflict resolve!!!

**Review by parkaransingh — APPROVED** [review]
Great job on figuring out auth!!!

**Pull request merged**